api/src/wfl/module/xx.clj

```diff
@@ -136,12 +141,16 @@
 (defmethod workloads/load-workload-impl
   pipeline
   [tx {:keys [items] :as workload}]
-  (let [unnilify     (fn [x] (into {} (filter second x)))
-        load-inputs! #(util/deep-merge workflow-defaults (util/parse-json %))]
+  (letfn [(unnilify [m] (into {} (filter second m)))
+          (load-inputs! [workflow]
+            (util/deep-merge workflow-defaults (util/parse-json workflow)))
+          (unpack-options [m]
+            (update m :workflow_options #(util/parse-json (or % "{}"))))]
     (->> (postgres/get-table tx items)
-      (mapv (comp #(update % :inputs load-inputs!) unnilify))
-      (assoc workload :workflows)
-      unnilify)))
+         (mapv (comp #(update % :inputs load-inputs!) unnilify unpack-options))
+         (assoc workload :workflows)
+         unnilify
+         unpack-options)))
```

Comment:
Aren't the workflow options per workflow? Why are you unpacking them at the workload level too?

Reply:
This is something in the spec doc. workflow_options are indeed per workflow, but something Trevyn pointed out is that often they'd want the same options for a huge number of workflows. Thus we provide the option at the workload-level too to accommodate that. Of course, anything provided on an individual workflow will override. 

This is also why we store workflow_options in the database at both the per-workflow and per-workload level, too

Reply:
Is this repeated (comp unnilify unpack-options) to handle the "two levels"?

Reply:
Yeah, we unpack both for every workflow and for the workload as a whole